src/commands/cicdPipeline.ts

```diff
@@ -0,0 +1,69 @@
+import * as vscode from 'vscode';
```

Comment:
💡 Since your PR has griwn this far, I realised the commands in use i.e. `configure` and `browse` both are no longer coupled at all, so I have tried to just re-organise your code for you here, and put a PR to your fork.

* https://github.com/kanika1894/vscode-aks-tools/pull/1

If you like it, you can merge it to your fork and it should appear as part of this PR.

📓 Please also note: Add the reference to the **other** work item which needs to be deployed before this work in this PR as well please. I fee, under description will be a good place. 

Thanks heaps.